MapboxNavigation/NavigationViewController.swift

```diff
@@ -557,6 +606,97 @@ extension NavigationViewController: StyleManagerDelegate {
         mapView?.reloadStyle(self)
     }
 }
+// MARK: - TopBannerViewController
+
+extension NavigationViewController: TopBannerViewControllerDelegate {
+    public func statusView(_ statusView: StatusView, valueChangedTo value: Double) {
```

Comment:
This method name doesn’t make it clear that the value refers to a speed multiplier.

Reply:
Technically, `statusView` doesn't have a concept of a speed multiplier, it just so happens that simulation multiplication is the only use-case of this delegate message. Should we change?

Reply:
(if we do, that might necessitate a deprecation.)

Reply:
This is now being tracked as an issue: https://github.com/mapbox/mapbox-navigation-ios/issues/2135